solr/config/configsets/oa_prod_20211208_v2/conf/lang/stemdict_nl.txt

```diff
@@ -0,0 +1,6 @@
+# Set of overrides for the dutch stemmer
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```

Comment:
Please move the TODO to the repository as the ticket. It'll be better to track.

Reply:
Just copied from the default schema, disregard these.